mne_connectivity/tests/test_connectivity.py

```diff
@@ -392,7 +392,11 @@ def test_metadata_handling(func, tmpdir, epochs):
     an ``mne.Epochs`` object input.
     """
     kwargs = dict()
-    if isinstance(epochs, np.ndarray) and func == spectral_connectivity_epochs:
+    if (
+        isinstance(epochs, np.ndarray)
+        and func == spectral_connectivity_epochs
```

Comment:
`and` is higher precedence than `or` so I think you want parentheses here? Or use `or func in (x, y)`